tests/distances/tanimoto.py

```diff
@@ -39,8 +48,78 @@ def _get_count_values() -> list[tuple[list[int], list[int], str, float, float]]:
     ]
 
 
+def _get_binary_matrices() -> list[
```

Comment:
I think that for bulk distances we can just use one test:
- use mols_list fixture like in other tests
- cut it to first 10 mols
- calculate ECFP fingerprints
- calculate regular similarities with double for loop and with bulk function
- verify that they are identical